src/components/[guild]/Requirements/components/GuildCheckout/components/buttons/ConnectWalletButton.tsx

```diff
@@ -1,23 +1,28 @@
-import { Collapse } from "@chakra-ui/react"
+import { Collapse, Tooltip } from "@chakra-ui/react"
 import useWeb3ConnectionManager from "components/_app/Web3ConnectionManager/hooks/useWeb3ConnectionManager"
 import Button from "components/common/Button"
-import { useAccount } from "wagmi"
 
 const ConnectWalletButton = (): JSX.Element => {
-  const { isConnected } = useAccount()
-  const { openWalletSelectorModal } = useWeb3ConnectionManager()
+  const { openWalletSelectorModal, type, isWeb3Connected } =
+    useWeb3ConnectionManager()
 
   return (
-    <Collapse in={!isConnected}>
-      <Button
-        size="lg"
-        colorScheme="blue"
-        onClick={openWalletSelectorModal}
-        isDisabled={isConnected}
-        w="full"
+    <Collapse in={type !== "EVM"}>
+      <Tooltip
+        label="Disconnect your Fuel wallet first"
+        isDisabled={!isWeb3Connected}
+        hasArrow
       >
```

Comment:
Yup, it should be visible, because currently we do not support any on-chain actions on Fuel, only on EVM chains, so this acts as a "Connect EVM wallet" button (maybe we should update the copy? 👀)

Reply:
Aah, I see, I don't think we need to change it, I was just making sure I understand the change 👍